packages/next-server/server/next-server.js

```diff
@@ -192,10 +192,9 @@ export default class Server {
     return sendHTML(req, res, html, req.method, this.renderOpts)
   }
 
-  async renderToHTML (req, res, pathname, query) {
+  renderToHTML (req, res, pathname, query) {
     try {
-      const out = await renderToHTML(req, res, pathname, query, this.renderOpts)
-      return out
+      return renderToHTML(req, res, pathname, query, this.renderOpts)
```

Comment:
```suggestion
      return await renderToHTML(req, res, pathname, query, this.renderOpts)
```

My bad, looking at the tests it seems like we need this because otherwise the try/catch is ignore / not used.

Reply:
Back to `return await renderToHTM()`

Thanks for looking Tim!